src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php

```diff
@@ -94,19 +83,16 @@ final class GetCustomerForViewingHandler implements GetCustomerForViewingHandler
     /**
      * @param TranslatorInterface $translator
      * @param int $contextLangId
-     * @param Link $link
      * @param Locale $locale
      */
     public function __construct(
         TranslatorInterface $translator,
         $contextLangId,
-        Link $link,
```

Comment:
Indeed this is a BC Break

Reply:
Yes, I will specify it in the PR description

Reply:
I reverted this BC break and kept the `link` for the next major.

Reply:
This is not a BC break as the query handler is instanciated by DI